db/migrate/20160126094510_add_badge_to_course_achievements.rb

```diff
@@ -0,0 +1,5 @@
+class AddBadgeToCourseAchievements < ActiveRecord::Migration
+  def change
+    add_column :course_achievements, :badge, :text
```

Comment:
By right, all achievements should have a badge, but I think this should be enforced by the tutors themselves (especially when you have designers working on the badge while a course is still being set up), rather than at the app/db level. 

The system can help handle achievements without badges through `default_url`.